src/scmtest/scitest.cljs

```diff
@@ -34,7 +35,10 @@
 
 (def res
   (try
-    (sci/eval-string code  {:env st :bindings (es/namespaces 'sicmutils.env)})
+    (sci/eval-string code
+                     {:env st
```

Comment:
`define-coordinates` is a macro which references symbols by namespace, so you also have to install the `:namespaces` key here to get everything working.